demo-app/src/index.js

```diff
@@ -11,22 +11,22 @@ const TOAST_LEVELS = ["info", "success", "error"];
 const TOP_ROUTE_EXTRACTOR = /^#?(\/[^\/]+).*/;
 const NAV_CONFIGS = [
   {
```

Comment:
apparently these include the merge from the parent too... Not sure If i can avoid that with git somehow?